slartibartfast/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link href='https://fonts.googleapis.com/css?family=Alex Brush' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Allura' rel='stylesheet'>
+    <title>Glimåkra Webbutvecklare Restaurangmenyer</title>
+
+</head>
+
+<body>
+
+    <header>
+        <section id="huvud" class="lös">
+            <img src="../lunchtimeweb-01_f73305aa-cd47-495d-9457-7a53dfcb022e_1024x1024@2x.png" alt="" width="200px">
+        <h1>Slartibartfast Kulinarer</h1>
+        <p></p>
+    </header>
+    <main>
+        <ul>
+            <li>
+                <a href="menyer/exempel/index.html">Menyer </a>
+            <li>
+                <h1>Meny</h1>
+            </li>
+            </li>
+        </ul>
+
+       <h3> <p>Välkommen till den kulinariska restauranger med de mest välutvecklade smaklökarna i denna galaxen. För en
+            extra upplevelse erbjuder vi även på poesi till eran rätt med den förfärligt dåliga ordbajsande Prostetnic
+            Vogon Jeltz</p></h3> <br>
+        </section>
+
+<section id="rätter" class="förvirrad"> 
+<div class="mat">
+
+        <ul>
+            <h3>Advanced Tea Substitute:</h3><i> almost, but not quite, entirely unlike tea</i></li> <br>
+            <h3>Algolian Zylat Burger:</h3><i>A kind of a burger made from the the most unpleasant parts of a creature
+                    well known for its total lack of any pleasant parts</i> </li> <br>
+            <h3>Starbix - the serious cereal:</h3> <i>simply nasty but nourishing</i> </li> <br>
+            <h3>Professional Sandwich from a professional sandwichmaker:</h3> <i>Made from a perfectly normal beast, quite tasty</i></li><br>
+           <h3>Pan Galactic Gargle Blaster:</h3> <i>Best drink in Existence, effect similar to having you brains
+                    smashed out by a slice lemon wrapped around a large gold brick</i></li><br>
+            <h3>Hagro Biscuit:</h3> <i>Blue bread similar to an iced finger, topped with guacamole and "other" stuff (Don't ask)</i></li> <br>
+            <h3>Old Janx Spirit:</h3><i>This drink cause your head to fly, your tongue will lie, your eyes fry and you might die</i></li><br>
+        </ul><br><br><br>
```

Comment:
Avoid using br tag for making space. Instead use CSS style (perhaps margin).
Also ul tag should only have li as children.